### ISSUE TITLE
[BUG] macOS 14 displays focusRing

### ISSUE THREAD

**Issue by peterpaulis** [fermoya/SwiftUIPager#329]
**Describe the bug**
on macOS a focus ring is shown as soon the user starts to interact with the pages

**To Reproduce**
- run on macOS 14
- start to swipe

**Expected behavior**
no focus ring should be shonw, or there should be the option to disable focusable and those keyboard control

**Screenshots / Videos**

https://github.com/fermoya/SwiftUIPager/assets/1956972/0fe71a96-d382-4b67-80f2-c447252c2c96

**Environment:**
 - OSX: macOS 14, XCode 15
- SwiftUIPager version: 2.5.0

**Additional context**

**Comment by peterpaulis** [comment]
if my pull request gets accepted, 

`.allowsKeyboardControl(false)`

can be used to disable keyboard control and the focus ring

**Comment by mayqiyue** [comment]
Great

**Comment by borut-t** [comment]
How can I apply this modifier @peterpaulis?
